framework/doc/sqa/SoftwareRequirements.tex

```diff
@@ -0,0 +1,454 @@
+
+\item \textit{Verification}. (1) The process of: evaluating a system or component to determine whether the products of a given development phase satisfy the conditions imposed at the start of that phase. (2) Formal proof of program correctness (e.g., requirements, design, implementation reviews, system tests). [ISO/IEC/IEEE 24765:2010(E) edited]
+\end{itemize}
+
+\subsection{Acronyms}
+
+\begin{tabular}{l l}
+  ASME    & American Society of Mechanical Engineers \\
+  DOE     & Department of Energy \\
+  FEM     & Finite Element Modeling \\
+  IEC     & International Electrotechnical Commission \\
+  IEEE    & Institute of Electrical and Electronics Engineers \\
+  INL     & Idaho National Laboratory \\
+  ISO     & International Organization for Standardization \\
+  IT      & Information Technology \\
+  M\&O    & Maintenance and Operations \\
+  MOOSE   & Multiphysics Object Oriented Simulation Environment \\
+  NQA     & Nuclear Quality Assurance \\
+  OOP     & Object Oriented Programming \\
+  QA      & Quality Assurance \\
+  V\&V    & Verification and Validation \\
+\end{tabular}
+
+
+\section{System Requirements}
+
+\begin{table}[!htbp]
+  \caption{Minimum Requirements.\label{tab:req}}
+  \begin{tabular}{|p{12cm}|}
+    \rowcolor{gray}
+    Minimum Requirements \\
+    \begin{itemize}
+    \item A POSIX compliant Unix including the two most recent versions of OS X and most current versions of Linux.
+    \item RAM: 4 GB for optimized compilation (8 GB for debug compilation), 2 GB per core execution
+    \item Disk: 100 GB
+    \item Compilers: GCC, Clang, and Intel
+    \item Python 2.6+
+    \item Git
+    \end{itemize} \\
+    \hline
+  \end{tabular}
+\end{table}
+
+
+\subsection{Functional Requirements}
+
+\begin{table}[!htbp]
+  \caption{Function Name: Core Framework, I/O and Execution Control.\label{tab:core}}
+  \newcounter{MyFirstTableCounter}
+  \setcounter{MyFirstTableCounter}{1}
+  \begin{tabular}{|l|p{12cm}|}
+    \rowcolor{gray}
+    Number & Requirement Description \\
+    F1.10 & The system shall allow support for user-defined object for controlling the execution stages of a simulation. \\ \hline
+    F1.20 & The system shall allow for user-defined time step selection for simulations using a time-based execution scheme. \\ \hline
+    F1.30 & The system shall allow for user-defined time step integration schemes for simulations using a time-based execution scheme. \\ \hline
+    F1.40 & The system shall support user-defined matrix preconditioning that may be applied during the solve stages. \\ \hline
+    F1.50 & The system should support a programatic method for building up the necessary objects necessary for a simulation. \\ \hline
+    F1.60 & The system shall provide the ability to resume a previous simulation due to fault or intentional termination. \\ \hline
+    F1.70 & The system shall allow for user-defined output types for simulation data. \\ \hline
+    F1.80 & Predictors \\ \hline
```

Comment:
"The system shall provide a method of providing improved initial guesses (also known as "predictions") for the solution at subsequent timesteps."